apps/demo/src/app/pages/notification/examples/03-close-notification/index.html

```diff
@@ -0,0 +1,7 @@
+<button
+    tuiButton
+    [disabled]="(notificationPermissionState$ | async) === 'denied'"
```

Comment:
And mappers to boolean :) `isGranted`, `isDenied`, `onlyGranted`?

Reply:
See this commit https://github.com/taiga-family/ng-web-apis/pull/123/commits/764b88fbf425ae110b34c719181d3c18983d5aca